dashboard/db/migrate/20160511210018_populate_hashed_email_in_users.rb

```diff
@@ -0,0 +1,14 @@
+# A migration that populates (if not already populated) a user's hashed_email
+# for users having an (non-empty) email address.
+
+class PopulateHashedEmailInUsers < ActiveRecord::Migration
+  def up
+    User.with_deleted.
+        where('email IS NOT NULL AND email <> ""').
+        where('hashed_email IS NULL OR hashed_email = ""').
+        find_each do |user|
+      user.hash_email
+      user.save!
+    end
+  end
```

Comment:
Do you want to add an empty down migration so that it is possible to roll back past this migration?  Seems harmless to do so.